R/string_operation.R

```diff
@@ -388,3 +388,31 @@ parse_character <- function(text, ...){
     readr::parse_character(text = text, ...)
   }
 }
+
+#' Wrapper function for readr::parse_number
+#' @param text to parse
+#' @export
+parse_number <- function(text, ...){
+  # readr::parse_number used to allow already numeric input, by doing nothing,
+  # but after updating readr version from 1.1.1 to to 1.3.1, it only allows character input.
+  # if numeric, return as is
+  if(is.numeric(text)) {
+    text
+  } else {
+    readr::parse_number(text = text, ...)
+  }
+}
+
+#' Wrapper function for readr::parse_logical
+#' @param text to parse
+#' @export
+parse_logical <- function(text, ...){
+  # After updating readr version from 1.1.1 to to 1.3.1, it only allows character input.
+  # So if the input is not character (e.g. numeric, Date, etc) either return as.is or NA based on the data type
```

Comment:
This commend does not seem to match what is actually done.

Reply:
Thank you for catching it. Updated comment.